pkg/kube/controllers/extendedstatefulset/util.go

```diff
@@ -12,6 +14,46 @@ import (
 	"code.cloudfoundry.org/quarks-utils/pkg/ctxlog"
 )
 
+// GetMaxStatefulSetVersion returns the max version statefuleset
+// of the extendedstatefulset.
+func GetMaxStatefulSetVersion(ctx context.Context, client crc.Client, exStatefulSet *estsv1.ExtendedStatefulSet) (*v1beta2.StatefulSet, int, error) {
+	// Default response is an empty StatefulSet with version '0' and an empty signature
+	result := &v1beta2.StatefulSet{
+		ObjectMeta: metav1.ObjectMeta{
+			Annotations: map[string]string{
+				estsv1.AnnotationVersion: "0",
+			},
+		},
+	}
+	maxVersion := 0
+
+	statefulSets, err := listStatefulSetsFromInformer(ctx, client, exStatefulSet)
+	if err != nil {
+		return nil, 0, err
+	}
+
+	ctxlog.Debug(ctx, "Getting the latest StatefulSet owned by ExtendedStatefulSet '", exStatefulSet.Name, "'.")
```

Comment:
Hm, maybe the resulting version number would be more interesting to log?

Reply:
Done.